docs/hw/mechanical.md

```diff
@@ -10,13 +12,16 @@ IMPORTANT: Always remove the Create® 3 robot’s battery prior to dismantling,
 ![The Create® 3 Top Drawing](data/top_drawing.svg "Create® 3 Top Drawing")
 ### Side View
 ![The Create® 3 Side Drawing](data/side_drawing.svg "Create® 3 Side Drawing")
+<sub>Wheel travel is approximately 30 mm.</sub>
 ### Bottom View
 ![The Create® 3 Bottom Drawing](data/bottom_drawing.svg "Create® 3 Bottom Drawing")
 ### Cargo Bay - Top
 ![The Create® 3 Cargo Bay Drawing](data/cargo_bay_drawing.svg "Create® 3 Cargo Bay Drawing")
 
 ## Downloadable 3D Model
```

Comment:
What do you think about putting the 3D model above the dimensioned drawing? It doesn't matter either way but I feel like it might look nicer.

Reply:
I moved it up and think it works quite well further up the page. Let me know if you disagree.